scripts/markdown_seo_lint.js

```diff
@@ -0,0 +1,219 @@
+const fs = require("fs");
+const yaml = require("js-yaml");
+const markdownlint = require("markdownlint");
+const path = require("path");
+
+const FRONT_MATTER_REGEX = /((^---\s*$[^]*?^---\s*$)|(^\+\+\+\s*$[^]*?^(\+\+\+|\.\.\.)\s*$))(\r\n|\r|\n|$)/m;
+
+function checkPageTitle(title) {
```

Comment:
Since we aren't using a strongly typed language, we need to be super clear in comments what the expected semantics of each function is. So I'd strongly urge you to put a doc comment on top of every function explaining what it does and what its return type is.

e.g. does `checkPageTitle(title)` take an object? A string? Does it throw an error if it encounters a problem? Or just return a user-friendly error message or `null`?